source/includes/_claim_fund_api.md

```diff
@@ -0,0 +1,1126 @@
+
+We also have a resend callback feature which you can read about [here](https://docs.oyindonesia.com/#feature-resend-callback-sending-payments). 
+
+### Callback Parameters
+
+| Parameter | Data Type | Description |
+| :---- | :---- | :---- |
+| tx\_status | String | Claim Fund Transaction Status |
+| tx\_status\_description | String | Description of the Claim Fund Transaction Status |
+| amount | BigInteger | Amount to be claimed |
+| partner\_trx\_id | String | Unique Transaction ID for a specific request, generated by the Client |
+| recipient\_email | String | Recipient Email to receive the Claim Fund Link |
+| recipient\_name | String | Recipient Name |
+| note | String | Claim Fund Notes. Accepts alphanumeric characters. |
+| claim\_fund\_link | String | Claim Fund Link which will be used to fill in Bank Account information |
+| expiration | String | Expiration for the Claim Fund Link. Using yyyy-MM-dd HH:mm:ss format in GMT+0 time zone. |
+| created | String | Claim Fund Link creation datetime. Using yyyy-MM-dd HH:mm:ss format in GMT+0 time zone. |
+| updated | String | Claim Fund last update datetime, based on the latest status update. Using yyyy-MM-dd HH:mm:ss format in GMT+0 time zone. |
+
+## Transaction Status
+Below is the list of transaction status of Claim Fund transaction:
+
+| Transaction Status <br> (Parameter: tx\_status) | Description <br> (Parameter: tx\_status\_description) | Details |
+| :---- | :---- | :---- |
+| INITIATED | The Claim Fund link has successfully been created and is awaiting the end user's response | State: Non-Final <br> The Claim Fund Link has been successfully created and sent to the Recipient’s Email. The Recipient needs to fill in the Bank Account information in the Claim Fund Link before the expiration date. |
+| EXPIRED | The Claim Fund link has expired or has been voided | State: Final <br> There are 2 cases: (1) The Claim Fund Link has expired because the Recipient did not fill in the Bank Account information before the expiration date occurred. (2) The Claim Fund Link has been cancelled or voided by the Owner before the expiration date occurs and the Recipient did not fill in the Bank Account information. |
+| WAITING\_APPROVAL | The Claim Fund transaction is awaiting approval | State: Non-Final <br> The Recipient has successfully filled in the Bank Account information and is waiting for the Owner’s approval to carry out the disbursement process. |
+| IN\_PROGRESS | The Claim Fund transaction is being processed | State: Non-Final <br> The Owner has approved the transaction and the disbursement process is in progress. |
+| PENDING | The Claim Fund transaction is currently awaiting further processing or confirmation | State: Non-Final <br> The Claim Fund Transaction did not receive a clear response from the Bank Network. The transaction is waiting for further processing or confirmation with the Bank Network.  |
```

Comment:
yg PENDING tampaknya ga perlu dimasukin since klo Pending, status Claim Fundnya tetep ditampilkan sbg IN PROGRESS? @guntoroyk